### PR TITLE
230 cw cert

### PR DESCRIPTION
**Pull request by Goncharo**
Ref: #metriport/metriport-internal#230

### Dependencies

N/A

### Description

get cert runner working end-to-end

### Release Plan

asap

### CODE REVIEW

**Review by Orta21 — APPROVED** [review]

**Review by leite08 — APPROVED** [review]

File: packages/packages/commonwell-cert-runner/src/document-contribution.ts [hunk 1]
```diff
@@ -130,5 +154,356 @@ async function getOrCreateOrg(
...
...W9vZENvZGU9IkVWTiI+DQoJCQkJCQkJCQk8dGVtcGxhdGVJZCByb290PSIyLjE2Ljg0MC4xLjExMzg4My4xMC4yMC4yMi40LjI3IiBleHRlbnNpb249IjIwMTQtMDYtMDkiLz4NCgkJCQkJCQkJCTxpZCByb290PSI0Y2U2MDQ2Yy1mNmUzLTQxYjAtOTFmYy0yZDUzMjVmMmJiYzMiLz4NCgkJCQkJCQkJCTxjb2RlIGNvZGU9IjI3MTAtMiIgY29kZVN5c3RlbT0iMi4xNi44NDAuMS4xMTM4ODMuNi4xIiBjb2RlU3lzdGVtTmFtZT0iTE9JTkMiIGRpc3BsYXlOYW1lPSJPWFlHRU4gU0FUVVJBVElPTiIvPg0KCQkJCQkJCQkJPHRleHQ+DQoJCQkJCQkJCQkJPHJlZmVyZW5jZSB2YWx1ZT0iI1NQTzJfMSIvPg0KCQkJCQkJCQkJPC90ZXh0Pg0KCQkJCQkJCQkJPHN0YXR1c0NvZGUgY29kZT0iY29tcGxldGVkIi8+DQoJCQkJCQkJCQk8ZWZmZWN0aXZlVGltZSB2YWx1ZT0iMjAxNDEwMDExMDMwMjYtMDUwMCIvPg0KCQkJCQkJCQkJPHZhbHVlIHhzaTp0eXBlPSJQUSIgdmFsdWU9Ijk4IiB1bml0PSIlIi8+DQoJCQkJCQkJCQk8YXV0aG9yIHR5cGVDb2RlPSJBVVQiPg0KCQkJCQkJCQkJCTx0aW1lIHZhbHVlPSIyMDE0MTAwMTEwMzAyNi0wNTAwIi8+DQoJCQkJCQkJCQkJPGFzc2lnbmVkQXV0aG9yPg0KCQkJCQkJCQkJCQk8aWQgZXh0ZW5zaW9uPSI1NTU1NTU1NTU1IiByb290PSIxLjEuMS4xLjEuMS4xLjIiLz4NCgkJCQkJCQkJCQk8L2Fzc2lnbmVkQXV0aG9yPg0KCQkJCQkJCQkJPC9hdXRob3I+DQoJCQkJCQkJCTwvb2JzZXJ2YXRpb24+DQoJCQkJCQkJPC9jb21wb25lbnQ+DQoJCQkJCQk8L29yZ2FuaXplcj4NCgkJCQkJPC9lbnRyeT4NCgkJCQk8L3NlY3Rpb24+DQoJCQk8L2NvbXBvbmVudD4NCgkJPC9zdHJ1Y3R1cmVkQm9keT4NCgk8L2NvbXBvbmVudD4NCjwvQ2xpbmljYWxEb2N1bWVudD4="
```
Comment (leite08) [inline packages/packages/commonwell-cert-runner/src/document-contribution.ts:428]: I know I approved this, but I didn't see this file as GH didn't show it bc it was too large.

Can we store this on a file? We have examples of file system usage with `fs` on this repo.

**Review by leite08 — CHANGES_REQUESTED** [review]

File: packages/packages/commonwell-cert-runner/src/document-contribution.ts (hunk 1 above)
Comment (Goncharo) [reply packages/packages/commonwell-cert-runner/src/document-contribution.ts]: put this in a file

File: packages/packages/commonwell-cert-runner/src/document-contribution.ts [hunk 2]
```diff
@@ -78,6 +93,15 @@ export async function documentContribution({
     console.log(respUpgradeLink);
   }
 
+  console.log(`>>> [E3] Populating test data on FHIR server...`);
+  const fhirApi = axios.create({
+    baseURL: fhirUrl,
+  });
+  const newPatientId = patientIdNewOrg.split("%5E%5E%5E")[0];
```
[the remark this answers is not in the document]
Comment (Goncharo) [reply packages/packages/commonwell-cert-runner/src/document-contribution.ts:101]: added as a TODO

File: packages/packages/commonwell-cert-runner/src/document-contribution.ts [hunk 3]
```diff
@@ -130,5 +154,365 @@ async function getOrCreateOrg(
     APIMode.integration
   );
 
-  return { orgAPI, orgName };
+  return { orgAPI, orgName, orgId: orgIdWithoutNamespace };
+}
+
+async function addOrgToFHIRServer(orgId: string, orgName: string, fhirApi: AxiosInstance) {
+  const data = `{
```
[the remark this answers is not in the document]
Comment (Goncharo) [reply packages/packages/commonwell-cert-runner/src/document-contribution.ts:163]: added as a TODO

**Pull request merged**